ip-masq-agent.yaml

```diff
@@ -14,7 +14,9 @@ spec:
       - name: ip-masq-agent
         image: gcr.io/google-containers/ip-masq-agent-amd64:v2.0.0
         securityContext:
-          privileged: true
+          privileged: false
+          capabilities:
+            add: ["NET_ADMIN", "NET_RAW"]
```

Comment:
I'm not sure why we will need `NET_RAW` but seems fine to start with this :)

Reply:
I'm not completely sure why yet either, but I did some looking around and every reference I've found specifies both for iptables.

Reply:
https://www.google.com/search?q=iptables+net_raw lots of examples, but nothing super authoritative. I've yet to try to find this in the kernel :upside_down_face: